skyplane/config.py

```diff
@@ -40,7 +40,7 @@
     "verify_checksums": True,
     "multipart_enabled": True,
     "multipart_min_threshold_mb": 128,
-    "multipart_min_size_mb": 8,
+    "multipart_min_size_mb": 5,  # 5 MiB minimum size for multipart uploads
```

Comment:
It is the minimum for AWS & GCP. For Azure, I think the minimum is just 0.

Reply:
I think this is being used to set the chunk size https://github.com/skyplane-project/skyplane/blob/07a1b8cb41c084ffbc2e4cf9173155c8bade5127/skyplane/replicate/replicator_client.py#L373 so we should separate the the minimum/default size. 

Do we have benchmarks on whats a good multipart chunk size? Anecdotally I remember seeing that larger chunk sizes (64-128MB) where better (maybe since it was better for parallelizing object store reads, since we do that pretty inefficiently atm) so defaulting to 5MB might be too small.

Reply:
Would something like 16 or 32MB be better? Both 5 and 8 seem too small. AWS recommends 100MiB as the minimum size threshold but doesn't set a default size per chunk. https://docs.aws.amazon.com/AmazonS3/latest/userguide/mpuoverview.html

Reply:
I'm currently benchmarking the performance with various chunk sizes, will you let know the results.

Reply:
Here are the results of a quick benchmark with a transfer of 100 files of size roughly 140 MB. 
|Chunk Size| Time | Throughput |
| --- | --- | ---- |
| 5 MiB | 173.116 s | 0.65 Gbps |
|8 MiB| 157.37s | 0.71Gbps |
|16 MiB| 23.69s | 4.72Gbps |
|32 MiB| 23.84s | 4.69Gbps |
|64 MiB | 17.03s | 6.56Gbps |
| 128 MiB | 19.96s | 5.60Gbps |

Based on this, 64 MiB may be a good choice for now.

Reply:
Great, let's go with 64MiB and ship it. In another PR, can you add a small documentation section to https://skyplane.org/en/latest/benchmark.html with this table and the high-level stats for this benchmark? This is very interesting.